app/odns.ml

```diff
@@ -66,32 +67,47 @@ let for_all_domains nameserver ~domains typ f =
           (Lwt_list.iter_p
              (fun domain ->
                 Udns_client_lwt.getaddrinfo t typ domain
+                >|= Rresult.R.reword_error
+                  (function `Msg msg as res ->
+                     Logs.err (fun m ->
+                         m "Failed to lookup %a for %a: %s\n%!"
+                           Udns_enum.pp_rr_typ (Udns.Rr_map.k_to_rr_typ typ)
+                           Domain_name.pp domain msg) ;
+                     res)
                 >|= f domain)
              domains) with
   | () -> Ok () (* TODO catch failed jobs *)
 
+let pp_response typ domain = function
+  | Error _ -> ()
```

Comment:
hnm, now there's no more output when an error (lookup failure) is encountered... is this intentional?

Reply:
> am confused by is the WARNING stuff that appears above the error

this is part of the [convenience function `is_reply`](https://github.com/roburio/udns/blob/aeee951f487cf8aa9b20720cc4860a9c3fa24b70/src/udns.ml#L2560) when invoked without `~not_error:false`. Maybe the `rcode` checking should not be done in there if this is confusing (I had to disable that in the server components as well)... maybe `Packet.is_reply` should do less work, not log, but instead return a result...